src/components/Link.tsx

```diff
@@ -7,6 +7,7 @@ export interface LinkProps extends MuiLinkProps {
 const Link = (props: LinkProps): JSX.Element => {
```

Comment:
Could destructure props here like
```suggestion
const Link = ( { sx, ...props }: LinkProps): JSX.Element => {
```